pywr/core.py

```diff
@@ -428,6 +430,17 @@ def load(cls, data, model=None, path=None, solver=None):
         # data is a dictionary, make a copy to avoid modify the input
         data = copy.deepcopy(data)
 
+        # check minimum version
+        try:
+            minimum_version = data["metadata"]["minimum_version"]
+        except KeyError:
+            pass
```

Comment:
That should encourage users to add it. We'll have to modify all the test models.